app/konnect/org-management/okta-role-mapping.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Map Konnect Roles to Okta Groups
+no_version: true
+badge: enterprise
+---
+By mapping Okta groups to [Konnect roles](/konnect/org-management/users-and-roles),
+you can manage user authorization directly through your Okta app.
+
+After mapping is set up:
+* Okta users belonging to the mapped groups can log into Konnect.
+* When a user logs into Konnect with their Okta account for the first time,
+Konnect will automatically provision an account with the relevant permission.
+* As a Konnect org admin, you will be able to see those users in Konnect,
+but you won't be able to edit them from the Konnect side. To manage
+automatically-created users, adjust user permissions through Okta.
+* If your org already has non-admin Konnect users before mapping, they will be
+mapped to the Okta application.
```

Comment:
I would change this to say "If your org already has non-admin Konnect users before mapping, on their next login they will be mapped to the roles defined by their Okta group membership."